gen3/docs/gen3-resources/developer-guide/architecture-new.md

```diff
@@ -0,0 +1,74 @@
+
+This documentation is intended for developers who want to understand the design and architecture of Gen3.
+
+##Overview
+
+Gen3 is a modular, open-source software platform that exposes a standard set of application programming interfaces (APIs) and user interfaces (UIs). It is capable of managing various types of cloud-based data and enabling cloud-based compute over those data. The underlying software and APIs are designed from the ground up to be interoperable, standards-based, and configurable.  The Gen3 software is cloud agnostic and its deployment is facilitated by containerization and orchestration frameworks such as Kubernetes.
```

Comment:
I added a sentence "While cloud-agnostic, we use AWS internally and therefore some documentation may include AWS specific instructions" which maybe covers your point?  I'm trying not to walk away from "cloud-agnostic" since it appears in a lot of places.

Reply:
That's probably fine, but it's not fully accurate. We have service(s) that are coupled with AWS services and won't work elsewhere. For example, the https://github.com/uc-cdis/ssjdispatcher relies on an AWS specific queue service. You can't run this in Google, and it's required if you're using the data upload flow through Fence (which, to be fair, we want to deprecate, but I think it's still used).  

That might actually be the only case 🤔